src/Components/shared/HeaderButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { StyleSheet, Text, TouchableOpacity } from 'react-native';
+import theme from '../../styles/theme.style';
+
+const HeaderButton = props => {
+  return (
+    <TouchableOpacity onPress={props.onPress} testID="headerButton">
+      <Text style={styles.headerButtonText}>✓</Text>
```

Comment:
And if we change it by an icon ? 
For instance, for the favorite Button ?
We could either set a prop Component or use `{props.children}` or ...